.gitignore

```
@@ -13,6 +13,8 @@ usage-data-config.json
 /cypress/screenshots
 /cypress/temp
 /cypress/videos
+/cypress/fixtures/extensions/extension-bar/package-lock.json
+/cypress/fixtures/extensions/extension-foo/package-lock.json
```

This could probably be simplified with

```suggestion
/cypress/fixtures/extensions/*/package-lock.json
```

---

Done